lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -491,6 +491,10 @@ func dockerContainerCreateParamsToPortlayer(cc types.ContainerCreateConfig, laye
 	config.WorkingDir = new(string)
 	*config.WorkingDir = cc.Config.WorkingDir
 
+	// attach
+	config.Attach = new(bool)
```

Comment:
config.Attach = swag.Bool(cc.Config.AttachStdin || cc.Config.AttachStdout || cc.Config.AttachStderr) - this should make things a little more clear. I spent several seconds trying to figure out what's going on. I also wonder if there are any references to this value from any other place.